sql_metadb/derived_tables/agreements_subscription_agreement_org_ext.sql

```diff
@@ -8,14 +8,15 @@ SELECT
     sao.sao_owner_fk AS subscription_agreement_id,
     sao.sao_org_fk AS sao_org_id,
     org.org_name AS sao_org_name,
-    sao.sao_role AS sao_role_id,
-    saor.rdv_value AS sao_role_value,
-    saor.rdv_label AS sao_role_label,
+	saor.saor_role_fk AS sao_role_id,
```

Comment:
I think `saor_role_fk` is as well only used within `mod_agreements` so we could leave the type varchar. 
Or did you just wanted a general clarification? As far as I remember we agreed that we only cast IDs to UUIDs that have their source outside of agreements to prevent incompatibilities.

Reply:
That column has varchar UUIDs in an unusual format and was not recognized by Metadb.  I have fixed it manually so that it is now a uuid type.  Metadb's conversion of varchar to uuid is not completed yet and we may see other cases like this.

Reply:
`saor_role_fk` is referencing to `rdv_id`  in `ref_data_value` which has varchar type at the moment. Changing `rdv_id`  to uuid type will temporary break a lot of the derived tables in ERM until all conversion has been finished.

Reply:
I will change it back.

Reply:
Thank you!